src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BORDER_NUMBER = 100;
+    private static final Random RANDOM = new Random();
+    private static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();

```

Comment:
`Random` and `ColorSupplier` are an instances of another object, so they are our dependencies and cannot be constant, so let's not name like they are, and make them non-static, we will learn how to work and initialize objects with dependencies later on
Constants can be primitives wrappers, or literals